pageserver/src/layered_repository.rs

```diff
@@ -1432,10 +1469,16 @@ impl LayeredTimeline {
             ancestor_timeline: ancestor_timelineid,
             ancestor_lsn: self.ancestor_lsn,
         };
-        let metadata_path =
-            LayeredRepository::save_metadata(self.conf, self.timelineid, self.tenantid, &metadata)?;
-        if let Some(relish_uploader) = &self.relish_uploader {
-            relish_uploader.schedule_upload(self.timelineid, metadata_path);
+
+        LayeredRepository::save_metadata(self.conf, self.timelineid, self.tenantid, &metadata)?;
+        if let Some(_relish_uploader) = &self.relish_uploader {
+            //     schedule_timeline_upload(LocalTimeline {
+            //         tenant_id: self.tenantid,
+            //         timeline_id: self.timelineid,
+            //         has_metadata: true,
+            //         layers: layer_uploads,
+            //         disk_consistent_lsn,
+            //     };
```

Comment:
This change just disables the upload, right? Is it intentional?

Reply:
Yes, due to:
* this PR being a subset of a bigger https://github.com/zenithdb/zenith/pull/686 one. I'm cutting that WIP one to separate pieces, so we could discuss and merge them in hopefully comprehensible bits
* uploads are disabled by default and while worked somehow before, they were not super good at it with weird threads they've spawned. IMO, no big issue to "break" them temporarily for sake of merging a big WIP pr eventually, that improves things

Reply:
Thanks. It would be nice to have some comment explaining that.

Reply:
Sure, I can do that, but I think it's better to be done in the thread model-related PR: https://github.com/zenithdb/zenith/pull/689

since more relevant and it has to do the same disabling, but in a bigger scale.
So, ping me when you get there, this is what we have to discuss eventually :slightly_smiling_face: